packages/react/src/utils/index.js

```diff
@@ -86,6 +86,29 @@ export const imageProxy = url =>
 
 export const isLazySupported = !isSSR && 'IntersectionObserver' in window
 
+export const formatSeconds = secs => {
```

Comment:
maybe we can use a browser API to have the same effect in less code?

Reply:
Hmm what do you have in mind @Kikobeats?